app/ide-desktop/lib/dashboard/src/hooks/debugHooks.ts

```diff
@@ -49,25 +49,28 @@ export function useDebugState<T>(
 // === useMonitorDependencies ===
 
 /** A helper function to log the old and new values of changed dependencies. */
-function useMonitorDependencies(
+export function useMonitorDependencies(
```

Comment:
Are we just tracking changes between renders, right?

Reply:
yes, for most of the debug hooks it logs the reason why the hook is being re-run.
it *is* used in `Debug.tsx` but is totally optional there and can be removed.

(or if you mean the documentation isn't quite descriptive enough then that can be improved too.)

Reply:
Btw did we include this hook into eslint aswell?

Reply:
which eslint rule? there is one for callback dependencies but that doesn't apply here as this hook doesn't accept a callback

Reply:
The same as for the `<Debug />` component: https://github.com/enso-org/enso/pull/10256#discussion_r1641885583